static/_redirects

```diff
@@ -576,3 +576,7 @@
 /getting-started/setup-prisma/start-from-scratch/mongodb/next-steps-* /docs/getting-started/setup-prisma/start-from-scratch/mongodb/next-steps
 /getting-started/setup-prisma/add-to-existing-project/relational-databases/next-steps-* /docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/next-steps
 /getting-started/setup-prisma/add-to-existing-project/mongodb/next-steps-* /docs/getting-started/setup-prisma/add-to-existing-project/mongodb/next-steps
+
+
+/accelerate/what-is-accelerate /docs/accelerate/
+/pulse/what-is-pulse /docs/pulse/
```

Comment:
recommend moving these to above the `Dynamic redirects` section. This is due to a weirdness with CF handling redirects.

Reply:
ah thanks for the hint, moved ✅